src/apps/chifra/internal/explore/output.go

```diff
@@ -25,18 +26,23 @@ import (
 // RunExplore handles the explore command for the command line. Returns error only as per cobra.
 func RunExplore(cmd *cobra.Command, args []string) (err error) {
 	opts := exploreFinishParse(args)
+	outputHelpers.InitJsonWriter("explore", &opts.Globals)
 	// EXISTING_CODE
 	// EXISTING_CODE
 	err, _ = opts.ExploreInternal()
```

Comment:
Can you please add a TODO: item here saying "TODO: does chifra explore need to use the StreamMany stuff? This is a sort of special case, but we can test it later.